db/migrate/20181201021612_create_effects.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
```

Comment:
Migrations make changes to the database. Each migration file name includes a timestamp for when the migration was created, and they normally will be run in the order of the timestamps.

The database tracks which migrations have already been run in a 'migrations' table, and after it runs will usually dump the schema to a schema.rb file. There's a rake task that will build the database from the current schema.rb, so if you are setting up a new database you don't have to run all of the migrations incrementally (since the schema will already include all the changes since the last time it was dumped). The schema file contains the last migration ran before the schema was dumped, and loading the database from the schema will populate the migrations table with all migrations up to the migration listed in the schema.

`bundle exec rake db:migrate` runs all migrations you are missing.
`bundle exec rake -T` will list all of the available 'rake tasks'. The one's to do with db/migration start with 'db:'. There are commands to rollback a migration, run a specific migration's up or down, etc.